plugins/trivy/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+# 0.28.2
```

Comment:
I think you need to use `##0.28.2` instead

Reply:
That was it!